src/wormhole/WormholeHelper.sol

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+/// library imports
+import "forge-std/Test.sol";
+
+/// bridge specific imports
+import "./lib/PayloadDecoder.sol";
+import "./lib/InternalStructs.sol";
+
+import {TypeCasts} from "../hyperlane/lib/TypeCasts.sol";
```

Comment:
I think we either move `TypeCasts` somewhere else, like a `common` directory, or have a duplicate in `wormhole/lib`, but not use the one in Hyperlane, makes a messy graph!